docs/strategies/fallback.md

```diff
@@ -65,6 +65,45 @@ new ResiliencePipelineBuilder<UserAvatar>()
 | `FallbackAction` | `Null`, **Required**                                                       | Fallback action to be executed.                                                             |
 | `OnFallback`     | `null`                                                                     | Event that is raised when fallback happens.                                                 |
 
+## Diagrams
+
+### Happy path sequence diagram
+
+```mermaid
+%%{init: {'theme':'dark'}}%%
+sequenceDiagram
+    actor C as Caller
+    participant P as Pipeline
+    participant F as Fallback
+    participant DM as DecoratedMethod
+
+    C->>P: Calls ExecuteAsync
+    P->>F: Calls ExecuteCore
+    F->>+DM: Invokes
+    DM->>-F: Returns result
+    F->>P: Returns result
+    P->>C: Returns result
+```
+
+### Unhappy path sequence diagram
+
+```mermaid
+%%{init: {'theme':'dark'}}%%
+sequenceDiagram
+    actor C as Caller
+    participant P as Pipeline
+    participant F as Fallback
+    participant DM as DecoratedMethod
```

Comment:
nit: just thinking, should we call this `DecoratedCallback` or `DecoratedUserCallback`? The same term as used in XML docs and our documentation.

Reply:
How about to add:

participant FA as FallbackAction

Reply:
Good point, let me change it